community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelTransactionImplementation.java

```diff
@@ -621,7 +623,7 @@ private void afterRollback()
     {
         try
         {
-            closeTransaction();
+            closeTransaction( -1 );
```

Comment:
`-1` should be a constant in the `CloseListener` than and it's name should be descriptive. It should also be used everywhere instead of `-1` including tests.